packages/hash/api/package.json

```diff
@@ -23,6 +23,7 @@
     "@aws-sdk/s3-presigned-post": "3.38.0",
     "@aws-sdk/s3-request-presigner": "3.38.0",
     "@blockprotocol/core": "0.0.8",
+    "@blockprotocol/type-system-web": "https://gitpkg.now.sh/blockprotocol/blockprotocol/packages/%40blockprotocol/type-system-web?c0a430d",
```

Comment:
I think we should temporarily just whitelist this domain for dependencies, with a TODO. We're in a dev branch 🤷